src/controllers/issuer.ts

```diff
@@ -503,6 +503,67 @@ export class IssuerController {
+	 *       400:
+	 *         $ref: '#/components/schemas/InvalidRequest'
+	 *       401:
+	 *         $ref: '#/components/schemas/UnauthorizedError'
+	 *       500:
+	 *         $ref: '#/components/schemas/InternalError'
+	 */
+	public async getResource(request: Request, response: Response) {
+		try {
+			if (request.params.did && request.params.resourceId) {
+				const didUrl = request.params.did+"?resourceId="+request.params.resourceId
+				const res = await new Identity(response.locals.customerId).agent.resolve(didUrl)
+
+				const contentType = res.headers.get("Content-Type")
+				const body = new TextDecoder().decode(await res.arrayBuffer())
+
+				return response.setHeader("Content-Type", contentType!).status(200).send(body);
+			} else {
+				return response.status(StatusCodes.BAD_REQUEST).json({
+					error: "The DID or resourceId parameter is empty."
+				})
+			}
```

Comment:
Fix the status codes, would be similar to what DID resolver returns so you could just bubble it up.

Reply:
Done! ✅